src/native/libs/System.Native/entrypoints.c

```diff
@@ -34,6 +34,7 @@
 static const Entry s_sysNative[] =
 {
     DllImportEntry(SystemNative_FStat)
+#if !defined(TARGET_APPLE) || defined(TARGET_OSX)
```

Comment:
src/native/libs are shared with Mono. How does this work on iOS - why are these changes not needed for Mono/iOS?

Reply:
This file is used only for the static entrypoint resolution in apphost. Technically we would not need it here either if I removed the (unused) apphost build.